hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -181,11 +181,13 @@
     CONSUMER_HTTP_CLIENT_VALIDATE_CERTS("consumer.http.client.validate.certs", true),
     CONSUMER_HTTP_CLIENT_VALIDATE_PEER_CERTS("consumer.http.client.validate.peer.certs", true),
     CONSUMER_HTTP_CLIENT_ENABLE_CRLDP("consumer.http.client.enable.crldp", true),
-
+    CONSUMER_HTTP_CLIENT_IDLE_TIMEOUT("consumer.http.client.idleTimeout", 30_000),
```

Comment:
I thought the proposition was to set 19 sec as a timeout.

Reply:
Yes but it is good only for us ;) We will configure it configuration files.